R/cookies.R

```diff
@@ -234,205 +234,206 @@ init_cookies <- function() {
     message("www folder already exists!")
   }
 
+  github_area <- "https://raw.githubusercontent.com/dfe-analytical-services/"
```

Comment:
I ended up copying this for the init_analytics code, so wondering if there's a better centralised way of doing this. e.g. a very simple non-exported function along the lines of:

```
github_area <- function(branch='main'){
   paste0(
      "https://raw.githubusercontent.com/dfe-analytical-services/dfeshiny/",
      branch,
      "/"
   )
}
```

Reply:
This would be neat (and we should do it in this PR if we stick with the data / files as they are), but feels like we're starting to bark up the wrong tree a bit if we need a function like this - see bigger comment

Reply:
Resolving to deal with separately in the other comments